applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/prj_release.conf

```diff
@@ -86,3 +86,41 @@ CONFIG_BT_CONN_TX_MAX=4
 CONFIG_SERIAL=n
 CONFIG_CONSOLE=n
 CONFIG_UART_CONSOLE=n
+
+################################################################################
+# DFU configuration
+
+CONFIG_SUIT=y
+CONFIG_SUIT_DFU_CANDIDATE_PROCESSING_MINIMAL=y
+CONFIG_ZCBOR=y
+CONFIG_ZCBOR_CANONICAL=y
+CONFIG_SSF_SUIT_SERVICE_ENABLED=y
```

Comment:
These options could potentially also be selected by the `DESKTOP_DFU_BACKEND_SUIT` option